transformers/avayaVoiceChannel/transformer.ts

```diff
@@ -0,0 +1,231 @@
+/** 
+ * Avaya voice channel
+ * version 0.1
+ */
+const COGNIGY_BASE_URL = 'https://endpoint-trial.cognigy.ai/';
```

Comment:
Please add a comment why the base url is required.

Reply:
Avaya CPaaS cmd "Gather" needs to know the url to pass the speech result to. In this case, the url is the full url of Cognigy's endpoint. It'd be better if Cognigy has API to get the current Cognigy's base url.